manifests/r/RescueTime,Inc/RescueTime/2.16.8.2/RescueTime,Inc.RescueTime.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RescueTime,Inc.RescueTime
+PackageVersion: 2.16.8.2
+PackageLocale: en-US
+Publisher: RescueTime, Inc.
+PackageName: RescueTime
+License: 2013

```

Comment:
A year or a copyright is not a license; Please provide the actual license terms for this package such as `Proprietary`, `MIT`, `GPL-2.0`, or similar

Reply:
Ok! This was autogenerated with `wingetcreate` and I actually didn't realize I was opening a pull request until it was already doing it 😅